llvm_jni/modules/jni.llvm/tests/testOutputs/arithmetic_no_wrap.ll

```diff
@@ -0,0 +1,6 @@
+define i64 @foo(i64 %0, i64 %1) {
+  %3 = add nsw i64 %0, %1
```

Comment:
Looks like this is adding just `nsw`. I think we want `nsw nuw`.  Can you get that from the C API?

Reply:
@jclark as far as I can work out C-API has only has [`LLVMBuildNSWAdd`](https://llvm.org/doxygen/group__LLVMCCoreInstructionBuilder.html#ga901f44572f6ef9f67e5f7bb496806280) and [`LLVMBuildNUWAdd`](https://llvm.org/doxygen/group__LLVMCCoreInstructionBuilder.html#gac46472d120e1a0d83222509e4e418ac2) nothing that can do both.